app/controllers/contributors_controller.rb

```diff
@@ -69,10 +71,12 @@ def conversations
   private
 
   def set_contributor
+    # TODO: scope by organization?
     @contributor = Contributor.find(params[:id])
   end
 
   def contributors_sidebar
+    # TODO: scope by organization
```

Comment:
https://github.com/tactilenews/100eyes/pull/1898/files#diff-1bd0a0fa1f76adfb12b7cc47fc7ab252b034c736ef7006a51b7353c10c581931R71-R74

Reply:
thnx ..done also in https://github.com/tactilenews/100eyes/pull/1987